docs/pages/identity-governance/entra-id.mdx

```diff
@@ -0,0 +1,359 @@
+}
+
+```
+
+</details>
+
+Your identity principal `<Var name="Principal ID" />` now has the necessary permissions to list Applications,
+Directories, and Policies.
+
+
+## Step 2/4: Generate configuration script (tctl)
+
+To begin integration, run the `tctl plugins install entraid` command.
+
+```code
+$ tctl plugins install entraid \
+    --name entra-id-default \
+    --auth-connector-name entra-id \
+    --default-owner=<Var name="Access List Owner"/> \
+    --default-owner=someOtherOwner@teleport.sh  \
+    --auth-server <Var name="example.teleport.sh:443" /> \
+    --use-system-credentials 
+
+Step 1: Run the Setup Script
+...
+
+Once the script completes, type 'continue' to proceed, 'exit' to quit: 
+```
+
+## Step 3/4: Configure Entra ID (tctl)
+
+(!docs/pages/includes/identity-governance/azure-shell.mdx!)
+
+## Step 4/4: Finish plugin installation (tctl)
+
+Once the script is done setting up the necessary permissions,
+it prints out tenant ID and client ID. Copy these values and provide it to the
+`tctl` to finish the integration.
+
+</TabItem>
+</Tabs>
+
+The integration is now configured and the Teleport Entra ID service will start 
+importing resources from Entra ID to Teleport.
+
+## Manual configuration
+
+Manual configuration may be preferable if you wish to have more control on the 
+Entra ID configuration. Below, an OIDC IdP based manual configuration is 
+demonstrated. 
+
+## Step 1/5. Create enterprise application
+
+In the Azure Portal, under “Azure services”, select “Enterprise applications”.
+Click on `+ New Application` button, then click `+ Create your own application` button.
+Enter a name for your application and create the application.
+
+![Entra ID Integration](../../img/igs/entraid/entra-create-enterprise-app.png)
+
+
+## Step 2/5. Configure SSO
+
+Open the newly created enterprise application.
+
+Under “Manage” menu select “Single sign-on”. In this configuration UI, you will need to set up Teleport
+as an SAML servicer provider.
+
+Click edit button to configure  “Basic SAML Configuration”.
+Enter the SAML assertion endpoint as Entity ID and ACS URL value.
+- **Entity ID and ACS URL:** SAML ACS endpoint of your Teleport cluster. 
+E.g. `https://<teleport cluster addr>/v1/webapi/saml/acs/entra-id`
+
+For “Attributes & Claims”, attributes with user will already be available for you but you will need to 
+add a `groups` claim. 
+
+![Entra ID Integration](../../img/igs/entraid/entra-sso-config.png)
+
+
+## Step 3/5. Configure OIDC IdP
+
+Under “App registrations” from Azure services menu, find and open your enterprise
+application created in step 1.
+
+Select “Manage > Certificates & secrets” and then select “Federated credentials“.
+Click `+ Add credential` button.
+
+Under “Add a credential” UI, configure credential with the following values:
+- **Federated credential scenario:** Other issuer
+
+Under “Connect your account”, configure the following values:
+- **Issuer:** https://example.com
```

Comment:
Someone reading these docs is likely to type the literal "example.com" value in here, which would be wrong. We need to better describe the steps we want the user to take.